lib/runtime/allocator.go

```diff
@@ -82,9 +82,11 @@ func (fbha *FreeingBumpHeapAllocator) growHeap(numPages uint32) error {
 	return nil
 }
 
-// Allocate determines if there is space available in WASM heap to grow the heap by 'size'.  If there is space
-//   available it grows the heap to fit give 'size'.  The heap grows is chunks of Powers of 2, so the growth becomes
-//   the next highest power of 2 of the requested size.
+// Allocate determines if there is space available in WASM heap to grow
+// the heap by 'size' bytes. If there is space available it grows the heap
+//  to fit the given 'size' number of bytes. The heap grows is chunks of
```

Comment:
```suggestion
// to fit the given 'size' number of bytes. The heap grows is chunks of
```

and move to another pr